app/config.py

```diff
@@ -42,6 +42,15 @@ class QueueNames(object):
     # further.
     DATABASE = "database-tasks"
 
+    # database operations for high priority notifications
+    PRIORITY_DATABASE = "-priority-database-tasks.fifo"
```

Comment:
Added the hyphen prefixes so that they match the queues we added in https://github.com/cds-snc/notification-terraform/pull/458, ie so they are named, for example,
`eks-notification-canada-ca-priority-database-tasks.fifo` 
rather than
`eks-notification-canada-capriority-database-tasks.fifo`
All our other SQS queues are not added in terraform - we should do that, and possibly at the same time rename the other queues as we have here (possibly moving the hyphen into the `NOTIFICATION_QUEUE_PREFIX`)